packages/darma-vt/package.py

```diff
@@ -210,10 +209,25 @@ def cmake_args(self):
             "-DUSE_STD_THREAD={}".format(
                 int(self.spec.variants["use_std_thread"].value)
             ),
-            "-Dvt_build_tests={}".format(int(self.spec.variants["build_tests"].value)),
-            "-Dvt_build_examples={}".format(
-                int(self.spec.variants["build_examples"].value)
-            ),
         ]
 
+        if self.spec.version == Version("develop"):
```

Comment:
If we do something like

```py
if self.spec.version > Version("1.3.0"):
```

Does spack make that comparison correctly?

If not we could try something like 
```py
if self.spec.version > Version("1.3.0") or self.spec.version == Version("develop"):
```

The reason for this is that when we (eventually) add our next release we won't have to worry about breaking the spack package.

Reply:
We should probably also add 1.1.0 and 1.2.0 to list of available versions. We currently only support 1.0.0 and develop:

https://github.com/DARMA-tasking/spack-package/blob/master/packages/darma-vt/package.py#L58-L59